app/views/layouts/application.html.erb

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <%=  stylesheet_link_tag 'feedback', integrity: true, crossorigin: 'anonymous' %>
+    <%=  stylesheet_link_tag "print.css", :media => "print", integrity: true, crossorigin: 'anonymous' %>
```

Comment:
Nitpick: this is using both single and double quotes, and new-style and old-style hashes. We should at least aim to be consistent on the line-level :)

Reply:
Oops, yes. Fixed.